vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -46,10 +46,21 @@
     shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/deploy/default/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
     when: CAPM3_VERSION == "v1alpha3"
 
+  - name: Ironic TLS setup and Ironic basic authentification sourcing
+    shell: "source ./{{ metal_dir }}/lib/{{ item }}"
```

Comment:
this would still not work. you are spawning a shell as a new process, sourcing the files, then the task is over, so is your shell. the next time you open a shell (in the next task), whatever you sourced is not there anymore. you should source in the same shell as where you are running what needs the sourced elements.

Reply:
Yes sorry I missed that. But how about other vars?

Reply:
Done